requirements.txt

```diff
@@ -1,5 +1,5 @@
 Pillow>=2.3.0,<=3.1.2
-numpy>=1.8.1,<=1.11.0
+numpy>1.8.1,<=1.11.0
```

Comment:
Why? If this is really needed, change it to `>=1.8.2` to match the syntax of the other lines (looks OK according to https://packages.ubuntu.com/search?keywords=python-numpy).